src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Figures;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figures getDefaultFigure() {
+        Figures circle = new Circle("WHITE", 10);
+        return circle;
+    }
+
+    public Figures getRandomFigures() {
+        int figuresIndex = new Random().nextInt(4);

```

Comment:
4 is a "magic number" - viewer of the code does not immediately understand why it is 4 here, and not 8, so let's use a constant
https://www.javatpoint.com/java-constant